src/components/sidebar/sidebar.tsx

```diff
@@ -119,7 +119,7 @@ export const isMainNavLinkActive = (href: string, path: string) => {
   )
 }
 
-const MainNavLink = ({ href, icon, children }: MainNavLinkProps) => {
+export const MainNavLink = ({ href, icon, children }: MainNavLinkProps) => {
```

Comment:
Do we need to export these? I don't see them being used anywhere.

Reply:
Relict from first draft, will remove.